src/routes/MemberProfile/MemberProfile.store.js

```diff
@@ -20,6 +20,7 @@ export function getRoleForCommunity (person, communitySlug) {
 export function presentPerson (person, selectedCommunitySlug) {
   return {
     ...person.ref,
+    projects: person.projects && person.projects.items,
```

Comment:
I wouldn't flatten the `items` pagination setup here. I think it could trip someone up later when/if there is need to implement the pagination on projects in this feature or elsewhere.